atom/browser/api/atom_api_web_contents.cc

```diff
@@ -1345,11 +1345,19 @@ bool WebContents::IsOffScreen() const {
 
 void WebContents::OnPaint(const gfx::Rect& dirty_rect,
                           const gfx::Size& bitmap_size,
+                          const int pixel_size,
```

Comment:
For primitive types there is no need to add `const`, since they are just copied, using `int` directly is preferred.